### ISSUE TITLE
Кобура ХоСу

### ISSUE THREAD

**Issue by silentenemy** [animusdev/green#303]
Запилить кобуру ХоСу, чтоб сбросить плащ, сумку и пойти строить офицериков. 
![http://www.t-line.ru/products_pictures/3ku3-b.jpg](Охуенная кобура)

**Comment by Euronem** [comment]
пояс плс

**Comment by ucnaHez** [comment]
К черту пояс.

Кобуры есть, но для ХоСа они не подходят по простой причине - у него нет пулевого оружия.

**Comment by fluorescent-net** [comment]
Имплаин еган нельзя засунуть в кобуру.
Это упущение, кстати.

**Comment by ucnaHez** [comment]
Чойта упущение? Еган здоровый.

**Comment by silentenemy** [comment]
Запилим хосу разгрузку или ременно-плечевую систему.

**Comment by fluorescent-net** [comment]
@ucnaHez сейчас бы размера ствола по пикселю измерять

**Comment by ucnaHez** [comment]
@fluorescent-net Ну, спрайт создает впечатление о предмете. Всегда считал, что еган больше, чем синдиревольвер, и уж тем более больше чем детективоревольвер и маленький пистолет синди.

**Comment by ucnaHez** [comment]
УЛЬТИМАТИВНОЕ РЕШЕНИЕ

Впилить ХоСу кобуру, выпилить ХоСу этот архиубогий респрайченный еган, вместо него впилить не столь убогий энержилюгер, который в билде есть, и позволить пихать оный в кобуру.

**Comment by ucnaHez** [comment]
Тока вот проблема.

У ХоСа на шее висят жетончики, а кобуру вешать некуда. Детективу-то жетончиков не полагается, у него с кобурой все окей.

Можно переделать механ жетончиков, чтобы они как-нибудь крепились прямо на тело, безо всяких слотов. Тогда их не будут снимать вместе со всей одеждой, и впилить, чтобы они оставались после испепеления и гиба. Тогда от них будет хоть какой-то толк - узнать, что этот высушенный труп - ХоС, или что тут коснулся решеточки и сгорел варден.
А снимать это через верб специальный.

Или можно впилить ХоСу предмет-гибрид кобуры и жетончика.

Или забить на кобуру ХоСу вообще.

**Comment by fluorescent-net** [comment]
> и впилить, чтобы они оставались после испепеления и гиба. Тогда от них будет хоть какой-то толк - узнать, что этот высушенный труп - ХоС, или что тут коснулся решеточки и сгорел варден.

Вот это хорошо и нужно. Исключение просто добавить, по сути. Совмещенный жетон и кобура - как-то совсем meh. Чёрт знает, что с этим делать.

**Comment by Strinkor** [comment]
@fluorescent-net Почему и нет? на кобуре будет написано, мол, эта кобура была презентована ХеСу "имяфамилия".

**Comment by ucnaHez** [comment]
Ето да.

Но кобура никак не может быть вещью, которая не сгорает и не взрывается, а суть жетончиков предполагается в этом.

**Comment by Strinkor** [comment]
@ucnaHez Да, допустим. А что если сделать что-то вроде импланта? Который медисканер будет показывать.

**Comment by SilentiumUA** [comment]
Короче говоря, энергетический маузер в кобуре подойдёт?
